### ISSUE TITLE
insane lag after closing a custom main level

### ISSUE THREAD

**Issue by heavyutils** [LatterRarity70/Main-Levels-Editor#50]
in the main level selector it gets very laggy after you closed a fake main level made with this

**Comment by user95401** [comment]
That is an actual issue?

**Comment by heavyutils** [comment]
Yeah. Though I can't rule out a conflict, but I think it's this one as it's only when I close level.
